apps/style/applab/style.scss

```diff
@@ -26,6 +26,13 @@ div#visualizationResizeBar {
   height: $workspace-headers-height;
 }
 
+// Override appliction.scss's defined width for applab, so that embed levels
+// get a better width. Non embed levels will be handled by responsive media
+// queries
+.small-footer-base {
+  width: 340px;
+}
+
```

Comment:
This would be cleaner if all the styles lived in the component (and we just did something like pass in a flag stating this was an embed view)